packages/smarthr-ui/src/components/Layout/stories/VRTStack.stories.tsx

```diff
@@ -0,0 +1,121 @@
+import { Meta, StoryObj } from '@storybook/react'
+import React from 'react'
+import styled from 'styled-components'
+
+import { Stack } from '../../..'
+import { Gap } from '../../../types'
+
+import { ColorBox } from './ComponentsForStories'
+
+/**
+ * gapはボリュームが多いため、1回ずつ出るように別途組み合わせます
+ * また、asも範囲が広いため、組み合わせていません。
+ * $ pict stack.pict.txt
+ * inline  align
+ * true    center
+ * false   baseline
+ * false   stretch
+ * false   start
+ * true    baseline
+ * true    flex-end
+ * false   end
+ * true    start
+ * true    flex-start
+ * true    end
+ * false   flex-end
+ * true    stretch
+ * false   center
+ * false   flex-start
+ */
+
+export default {
+  title: 'Layouts（レイアウト）/Stack',
```

Comment:
TabBar や Button など、整備済み Story と合わせてほしいです!

```suggestion
  title: 'Layouts（レイアウト）/Stack/VRT',
```

Reply:
気づいておりませんでした🙇‍♀️失礼しました
https://github.com/kufu/smarthr-ui/pull/5017/commits/a04236ffa0be17060343f7f45f882accafd97c50